shared/src/license.ts

```diff
@@ -100,6 +100,7 @@ export const taskIdLicenseNameMapping = {
   "ticket-broker-reseller-registration": "Ticket Brokers",
   "telemarketing-license": "Telemarketers",
   "entertainment-agency-reg": "Employment & Personnel Service-Entertainment/Booking Agency",
+  "cemetery-certificate": "Cemetery-Cemetery",
```

Comment:
[pebble] Are we following a naming convention with the `Cemetery-Cemetery` approach. Noticed that `pharmacy-pharmacy` has the same convention however `telemarketing-license` so wanted to point that out.

Reply:
@aadedejifearless - `cemetery-certificate` was the id given in the ticket.